gnd/src/main/java/com/google/android/gnd/ui/common/AbstractView.java

```diff
@@ -39,20 +38,19 @@ public AbstractView(@NonNull Context context, @Nullable AttributeSet attrs) {
     super(context, attrs);
   }
 
-  @Nullable
-  private FragmentActivity getActivity() {
+  protected FragmentActivity getActivity() {
     Context context = getContext();
     while (context instanceof ContextWrapper) {
       if (context instanceof FragmentActivity) {
         return (FragmentActivity) context;
       }
       context = ((ContextWrapper) context).getBaseContext();
     }
-    return null;
+    throw new NullPointerException("Activity is null");
```

Comment:
Not sure this is the right exception/error to report here. `IllegalStateException` is probably more appropriate, with a message like "View is not contained in FragmentActivity".